service/src/test/java/uk/gov/hmcts/reform/fpl/service/ReturnApplicationServiceTest.java

```diff
@@ -57,7 +52,7 @@ private ReturnApplication buildReturnedApplication() {
             .build();
     }
 
-    private DocumentReference buildDocumentReference() {
+    private DocumentReference applicationDocument() {
```

Comment:
is this meant to be buildApplicationDocument?

Reply:
can be, it returns the application document. Sometimes I put build sometimes I don't should probs be more consistent on that

Reply:
Only suggesting for consistency since there is the "buildReturnedApplication" function in the same file